cmd/create/cluster/cmd.go

```diff
@@ -1175,7 +1196,8 @@ func run(cmd *cobra.Command, _ []string) {
 	subnetIDs := args.subnetIDs
 	subnetsProvided := len(subnetIDs) > 0
 	r.Reporter.Debugf("Received the following subnetIDs: %v", args.subnetIDs)
-	if !useExistingVPC && !subnetsProvided && interactive.Enabled() {
+	// If the user has set the availability zones (allowed for non-BYOVPC clusters), don't prompt the BYOVPC message
+	if !useExistingVPC && !subnetsProvided && !isAvailabilityZonesSet && interactive.Enabled() {
```

Comment:
@zgalor @pvasant I've added ` && !isAvailabilityZonesSet` to the **if** statement.